simtools/visualization/visualize.py

```diff
@@ -714,3 +714,96 @@ def plot_array(
     plt.tight_layout()
 
     return fig
+
+
+def plot_simtel_ctapipe(filename, cleaning_args, distance, return_cleaned=False):
+    """
+    Reads in a simtel file and plots reconstructed photoelectrons via ctapipe.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the simtel file.
+    cleaning_args : tuple, optional
+        Cleaning parameters as (boundary_thresh, picture_thresh, min_number_picture_neighbors).
+    distance : astropy Quantity, optional
+        Distance to the target.
+    return_cleaned : bool, optional
+        If True, apply cleaning to the image.
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        The matplotlib figure containing the plot.
+    """
+    # pylint:disable=import-outside-toplevel
+    import numpy as np
+    from ctapipe.calib import CameraCalibrator
+    from ctapipe.image import tailcuts_clean
+    from ctapipe.io import EventSource
+    from ctapipe.visualization import CameraDisplay
+
+    default_cleaning_levels = {
+        "CHEC": (2, 4, 2),
+        "LSTCam": (3.5, 7, 2),
+        "FlashCam": (3.5, 7, 2),
+        "NectarCam": (4, 8, 2),
+    }
+
+    source = EventSource(filename, max_events=1)
+    event = None
+    for event in source:
+        print(event.index.event_id)
+    tel_id = sorted(event.r1.tel.keys())[0]
+
+    calib = CameraCalibrator(subarray=source.subarray)
+    calib(event)
+
+    geometry = source.subarray.tel[1].camera.geometry
+    image = event.dl1.tel[tel_id].image
+    cleaned = image.copy()
+
+    if return_cleaned:
+        if cleaning_args is None:
+            boundary, picture, min_neighbors = default_cleaning_levels[geometry.name]
```

Comment:
You are ok with these lines not covered by a test?

Reply:
Yes at the moment this is not really important.